File: ProgrammersGuide/WorkingWithProjects/FiscalYearProperties/FiscalYearPropertiesExamples.cs
using Helpers;
using NUnit.Framework;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Assert = NUnit.Framework.Assert;
using Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute;

namespace Examples.ProgrammersGuide.WorkingWithProjects.FiscalYearProperties
{
    [TestClass, TestFixture]
    public class FiscalYearPropertiesExamples
    {	
        [TestMethod, Test, Owner("Console")]
        public void ReadingFiscalYearProperties()
        {
            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/FiscalYearProperties/ReadingFiscalYearProperties");

            ReadingFiscalYearPropertiesExample.Program.Main();
        }

        [TestMethod, Test, Owner("Console")]
        public void WritingFiscalYearProperties()
        {
            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/FiscalYearProperties/WritingFiscalYearProperties");

            WritingFiscalYearPropertiesExample.Program.Main();
        }

    }
}

File: Examples/ProgrammersGuide/WorkingWithProjects/FiscalYearProperties/FiscalYearPropertiesExamples.vb
Imports Helpers
Imports NUnit.Framework
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports Assert = NUnit.Framework.Assert
Imports Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute

Namespace Examples.ProgrammersGuide.WorkingWithProjects.FiscalYearProperties
	<TestClass, TestFixture> _
	Public Class FiscalYearPropertiesExamples
        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub ReadingFiscalYearProperties()
	            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/FiscalYearProperties/ReadingFiscalYearProperties")
	
	            ReadingFiscalYearPropertiesExample.Program.Main()
        End Sub

        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub WritingFiscalYearProperties()
	            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/FiscalYearProperties/WritingFiscalYearProperties")
	
	            WritingFiscalYearPropertiesExample.Program.Main()
        End Sub

	End Class
End Namespace